### ISSUE TITLE
Loosing connection to own xmpp-server

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#350]
```
Hello,

GTalkSMS uses my own XMPP-Server and is loosing connection after few hours but 
the symbol in the notification bar is still green. To reestablish the 
connection, I have to press "disconnect" and "connect".

With a GTalk-account this problem doesn't exist.

Logfiles will come later.



```

Original issue reported on code.google.com by `vinzentt...@gmail.com` on 21 Jan 2014 at 12:43

**Comment by GoogleCodeExporter** [comment]
```
Additional informations:

GTalkSMS: 4.4.4
Android: 4.2.2
Device: Wiko Ozzy
```

Original comment by `vinzentt...@gmail.com` on 21 Jan 2014 at 12:47

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
And here are the logfiles


GTalkSMS Preferences
notifiedAddress: personalaccount@example.net
useGoogleMapUrl: true
cameraMaxFileSizeInMegaBytes: 0
manuallySpecifyServerSettings: true
batteryNotificationInterval: 20
startAtBoot: true
forceMucServer: true
showStatusIcon: true
notifyInMuc: false
notificationIncomingSmsType: both
notifyBatteryInStatus: true
ringtone: content://settings/system/ringtone
login: phoneaccount
stopOnPowerDisconnected: false
notifySmsDelivered: false
xmppBlockedResourcePrefixes: android
MessagingA
xmppSecurityMode: req
hiddenNotifications: passwort1
notifySmsSent: true
pingIntervalInSec: 600
roomPassword: passwort2
cameraProfile:
serverHost: xmpp.example.net
cameraRotationInDegree: 90
mucServer: muc.xmpp.example.net
notifyApplicationConnection: false
notifyBattery: true
locale: default
smsMagicWord: password3
publicIntentsEnabled: false
formatResponses: true
useOpenStreetMapUrl: false
serviceName: example.net
notificationIgnoreDelay: 1000
markSmsReadOnReply: true
useCompression: false
startOnBoot: false
displayIconIndex: 0
startOnPowerConnected: false
smsReplySeparate: true
hiddenMsgNotifications:
startOnWifiConnected: false
stopOnWifiDisconnected: false
debugLog: true
smsNumber: 5
connectOnMainscreenShow: true
publicIntentTokenRequired: false
blockedResourcePrefixes: android
MessagingA
cameraMaxDurationInSec: 0
serverPort: 5222
displayContactNumber: true
stopOnPowerDelay: 1
notifyIncomingCalls: true
publicIntentToken: secret

GTalkSMS Version: 4.4.4
Android API: 17
Kernel info: 3.4.5
android@HTSVR8) )
#1 SMP PREEMPT Fri Nov 22 14:45:42 CST 2013

--------- beginning of /dev/log/main
```

Original comment by `vinzentt...@gmail.com` on 21 Jan 2014 at 2:15

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
I tried now different possibilities to connect to my xmpp server. Through 3G 
(UMTS/HSPA), 2G (GPRS/EDGE) and WIFI. With all 3 possibilities there are 
connections losses. I now switched back to a gtalk-account.

We will see, if there is the same issue.
```

Original comment by `vinzentt...@gmail.com` on 22 Jan 2014 at 3:02

**Comment by GoogleCodeExporter** [comment]
```
No problems when using the google-account.
```

Original comment by `vinzentt...@gmail.com` on 23 Jan 2014 at 10:19

**Comment by GoogleCodeExporter** [comment]
```
Hello, sorry for the delay I'm quite busy outside of GTalkSMS.
Do you have enable the keepalive mechanism on your server? With which 
configuration?

Since few versions you can change the GTalkSMS ping interval inside the 
application.
Which XMMP-server are you using on which plateform?

Thanks
```

Original comment by `Florent....@gmail.com` on 27 Jan 2014 at 9:12

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 16 Aug 2014 at 11:40
- Changed state: **FixedInBeta**

**Comment by GoogleCodeExporter** [comment]
```
Released on Play Market as 5.0
```

Original comment by `Florent....@gmail.com` on 25 Oct 2014 at 6:23
- Changed state: **Deployed**
